utils/types.py

```diff
@@ -1,3 +1,23 @@
 from discord.ext import commands
+from typing import TypedDict
+
+
+class TimetableEntry(TypedDict):
```

Comment:
The types module  (in the testing PR renamed to `typeAliases`) is actually only meant for type aliases.

S: Place models in the `/models` directory. E.g. the mensa has models that are not in the database but still in `/models`.